chefctl/src/chefctl.rb

```diff
@@ -1238,6 +1245,15 @@ def parse_both_passes(argv = nil)
       options[:splay] = v.to_i
     end
 
+    parser.on(
+      '-t', '--trace',
+      'Enable chef trace logging. This is a shortcut to passing' +
+      '" -- -l trace" directly. This supersedes --debug'
```

Comment:
s/supersedes/implies/ maybe? meaning, does `-t` also result in the `-d` stuff being turned on?

Reply:
That's not really how log levels work.

If you set a log level to trace and then later set it to debug, it no longer includes trace.

Yes, `trace` level means `debug` messages (and `warn` and `info`) get through. But that's not really the point. The point is

-d does `-l debug` and `-t` does `-l trace`, and what happens if you do both of them is probably unclear, so we're letting you know if they're both set we drop `-d` and just act on `-t`.